themes/classic/templates/customer/addresses.tpl

```diff
@@ -29,13 +29,19 @@
 {/block}
 
 {block name='page_content'}
-  {foreach $customer.addresses as $address}
-    <div class="col-lg-4 col-md-6 col-sm-6">
-    {block name='customer_address'}
-      {include file='customer/_partials/block-address.tpl' address=$address}
-    {/block}
+  {if $customer.addresses}
+    {foreach $customer.addresses as $address}
+      <div class="col-lg-4 col-md-6 col-sm-6">
+      {block name='customer_address'}
+        {include file='customer/_partials/block-address.tpl' address=$address}
+      {/block}
+      </div>
+    {/foreach}
+  {else}
+    <div class="alert alert-info" role="alert" data-alert="info">
+      {l s='No addresses are available.' d='Shop.Notifications.Success'} <a href="{$urls.pages.address}" title="{l s='Add a new address.' d='Shop.Theme.Actions'}">{l s='Add a new address.' d='Shop.Theme.Actions'}</a>
```

Comment:
Sorry, but I hadn't noticed these wordings. It seems that "Add a new address." is a new string. The existing one doesn't have a period. Also, "No addresses are available" does exist, but with a variable (%) at the end?

Reply:
![image](https://user-images.githubusercontent.com/70583503/147254073-e80b8048-7922-4461-9a07-48a5178d5156.png)
![image](https://user-images.githubusercontent.com/70583503/147254109-ce6144ae-5c32-4f73-8ce8-0654400acf17.png)

Reply:
@Julievrz It's the same string, but separated to two. Should I use the variable replacement like it was in the controller @PierreRambaud?

Reply:
Separating the string creates a new one. If possible, that would be great to keep the existing one. 😇

Reply:
It should be a new string I think.
I see no reason to have ` %s` without using it. In the end it will add an empty space for nothing

Reply:
Ok if the variable is useless then we can create a new string. I thought Daniel was saying that the text and the variable had been separated into 2 strings instead of one.

Reply:
Well, it was basically replacing one sentence into another, it's better if it's two strings. ;)